src/nni_manager/training_service/remote_machine/sshClientUtility.ts

```diff
@@ -114,9 +112,6 @@ export namespace SSHClientUtility {
         // Decompress the remote compressed file in and delete it
         await remoteExeCommand(`tar -oxzf ${remoteTarPath} -C ${remoteDirectory}`, sshClient);
         await remoteExeCommand(`rm ${remoteTarPath}`, sshClient);
```

Comment:
why not `return Promise.resolve()`?

Reply:
no return value (undefined) is automatically wrapped as Promise<void> since `async` is used.

Reply:
i see